src/Twig/StorageExtension.php

```diff
@@ -7,14 +7,18 @@
 use Gaufrette\Extras\Resolvable\ResolverInterface;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFilter;
+use Symfony\Component\Asset\Packages;
 
 final class StorageExtension extends AbstractExtension
 {
     private $awsS3PublicUrlResolver;
+    private const AUTHORIZED_SCHEMA_REGEX = '/^(https?:\/\/)/';
```

Comment:
Move the const out of the other variables

Reply:
no problem but it's you who told us to place here

Reply:
Here as at the beginning of the class, but usually we do not mix variables, static variables and constants for readability